[2025-08-24T20:15:39.202+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl.transform_new_york manual__2025-08-24T20:10:34.878611+00:00 [queued]>
[2025-08-24T20:15:39.213+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl.transform_new_york manual__2025-08-24T20:10:34.878611+00:00 [queued]>
[2025-08-24T20:15:39.215+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-08-24T20:15:39.230+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_new_york> on 2025-08-24 20:10:34.878611+00:00
[2025-08-24T20:15:39.235+0000] {standard_task_runner.py:60} INFO - Started process 274 to run task
[2025-08-24T20:15:39.240+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_etl', 'transform_new_york', 'manual__2025-08-24T20:10:34.878611+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpv_4q4ors']
[2025-08-24T20:15:39.244+0000] {standard_task_runner.py:88} INFO - Job 71: Subtask transform_new_york
[2025-08-24T20:15:39.337+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_etl.transform_new_york manual__2025-08-24T20:10:34.878611+00:00 [running]> on host 6db65bba76ed
[2025-08-24T20:15:39.424+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl' AIRFLOW_CTX_TASK_ID='transform_new_york' AIRFLOW_CTX_EXECUTION_DATE='2025-08-24T20:10:34.878611+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-24T20:10:34.878611+00:00'
[2025-08-24T20:15:39.447+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-24T20:15:39.458+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_etl, task_id=transform_new_york, execution_date=20250824T201034, start_date=20250824T201539, end_date=20250824T201539
[2025-08-24T20:15:39.494+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-24T20:15:39.519+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
